src/wp-admin/js/media-upload.js

```diff
@@ -1,13 +1,36 @@
-/* global tinymce, QTags */
-// send html to the post editor
+/* global wpActiveEditor, tinymce, QTags */
+
+/**
+ * Updates the ThickBox anchor href and the ThickBox's own properties in order to set the size and position on every resize event.
+ * Also adds a function to send HTML or text to the currently active editor.
+ *
+ * @summary Makes sure the ThickBox dimensions are properly set and adds functionality to pass HTML to the editor.
+ *
+ * @since 2.3.3
+ *
+ * @requires jQuery
+ */
+
 
 var wpActiveEditor, send_to_editor;
 
+/**
+ * @summary Sends the HTML passed in the parameters to TinyMCE.
+ *
+ * @since 2.3.3
+ *
+ * @global {tinymce} tinymce An instance of the TinyMCE editor.
+ * @global {QTags} QTags An instance of QTags.
+ *
+ * @param {string} html The HTML/text to be sent to the editor.
```

Comment:
Can we assume it is either HTML or text that is sent? Isn't this always treated as HTML from and to the editor?